src/bacnet/basic/object/ai.c

```diff
@@ -49,6 +49,7 @@
 #endif
 
 static ANALOG_INPUT_DESCR AI_Descr[MAX_ANALOG_INPUTS];
```

Comment:
what type is ANALOG_INPUT_DESCR ? char* ?

Reply:
44 
 45     typedef struct analog_input_descr {
 46         unsigned Event_State:3;
 47         float Present_Value;
 48         BACNET_RELIABILITY Reliability;
 49         bool Out_Of_Service;
 50         uint8_t Units;
(...)
} ANALOG_INPUT_DESCR;